packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1037,7 +1037,15 @@ class RenderBackdropFilter extends RenderProxyBox {
   void paint(PaintingContext context, Offset offset) {
     if (child != null) {
       assert(needsCompositing);
-      context.pushLayer(BackdropFilterLayer(filter: _filter), super.paint, offset);
+      final PaintingContextCallback paintingCallback = (PaintingContext context, Offset offset) {
```

Comment:
Can't this be statically defined? Instead of instantiating a new closure on each paint call.

Reply:
If I make a static, I can no longer call `super.paint` which is non-static?

Don't worry, I'll remove this hacky callback soon. See my 3-step plan in https://github.com/flutter/flutter/issues/29070#issue-418990863

Reply:
I meant make this function statically defined, not make this the function static. 

e.g.

```dart
void paintingCallback(PaintingContext context, Offset offset) {
  blah
}

void paint(PaintingContext context, Offset offset) {
  context.pushLayer(BackdropFilter(filter: _filter), paintingCallback, offset);
}
```

Reply:
Oh, great, that's a good idea. Thanks!